eng/templates/stages/release.yml

```diff
@@ -390,7 +390,11 @@ stages:
               target_branch="${target_branch#internal/}"
             fi
           fi
-
+          
+          # The resolved variables will be utilized to update the Versions.props file within the submit-source-build-release-pr script.
+          resolvedSourceBuiltArtifactsFileName=$(basename $(PIPELINE.WORKSPACE)/$(sourceBuiltArtifactsFileName))
```

Comment:
Technically, you should use double-quotes to enclose `$(PIPELINE.WORKSPACE)/$(sourceBuiltArtifactsFileName)` so that the script doesn't fail when you have paths with spaces. It's unlikely to happen, but it's a good practice to think about this.

Reply:
Done, thank you :)

Reply:
Perhaps I am missing something. However, I am not obtaining the desired outcome in this manner. 
"$(PIPELINE.WORKSPACE)/$(sourceBuiltArtifactsFileName)")
https://dev.azure.com/dnceng/internal/_build/results?buildId=2186015&view=logs&j=19992227-62fb-5b50-4e29-3b72bd33eea1&t=9281c869-17dc-5ef7-6bc6-1c1da071debd&l=103
it returns: Private.SourceBuilt.Artifacts.*.centos.8-x64.tar.gz